terraform/cloudsql/postgresql/bind/versions.tf

```diff
@@ -9,5 +9,10 @@ terraform {
       source  = "hashicorp/random"
       version = ">=3.1.0"
     }
+
+    local = {
+      source = "hashicorp/local"
+      version = "2.2.2"
```

Comment:
I think this should be expressed as a range rather than an exact version. This is because when a new version comes out, we will bump the version in the manifest, and this will start to fail.

Reply:
Makes sense